notebooks/introduction_to_tensorflow/solutions/2a_dataset_api.ipynb

```diff
@@ -248,12 +248,17 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>"This function includes many convenient features, so the data is easy to work with. This includes:"</li></ul><p>--&gt; "This function has many convenient features:"</p><p> </p><p> </p><ul><li>This function returns --&gt; It returns</li></ul><p> </p><ul><li>What do mean by "Using the column headers as dictionary keys" ? Rephrase maybe.</li></ul><p> </p><ul><li>"Automatically determining the type of each column." --&gt; "It automatically determines the type of each column."</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>